install_config/master_node_configuration.adoc

```diff
@@ -1261,6 +1261,12 @@ rules:
 
   # A catch-all rule to log all other requests at the Metadata level.
   - level: Metadata <1>
+
+  # Log login failures from the web console or CLI. Turn on the audit, then create policy based on the requested data.
```

Comment:
If both methods are required or encouraged, it should be "webconsole and CLI" 

 "Turn on the audit, then create policy based on the requested data." doesn't fit here. I might change it to a note to review the logs and refine your policies, as required, around line 1187.

Reply:
`or` is because you can log either, each rule applies to a different element. Let's be explicit about it.